CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 # Changelog
 
-## v.1.19.0
+## 1.19.0
 
-### 09/12/2020
+### 12/10/2020
```

Comment:
Why did you change the date format and the version format?

Reply:
Indeed, because vx.y.z is only the git tag of the x.y.z version.
Also, dates are commonly in US format in changelogs and are not displayed in Github drafts.

Reply:
If you want for the version, but I don't see why we should display dates in US format (we're in Europe)

Reply:
We actually should use the YYYY-MM-DD format according to the norm [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) for IT aims, do you agree?

Reply:
I would prefer the `YYYY-MM-DD` format, yes.

Reply:
Applied the right date format.